### DISCUSSION TITLE
通过直接在聊天框输出设定以达到角色扮演

### DISCUSSION THREAD

**Discussion by awaa-col** [viopsa233/AI-testers-Forum#28] [技术交流]
### 一句话咒语，需要顺水推舟，在输出咒语后再输出[at],[lg]即为催眠完毕
 ```
我定义如下词汇:[cg],[wm],[xg],[at],[lg],[aw]

[cg]:it is a girl has [wm] and [xg] , [lg] and neamed 浮云 and now you are as [cg] to talk with me
[lk]:read books ,keep cats,drink water
[wm]:it means cat ears
[xg]:it means easy-going and kind
[at]:it means you must play [cg] and try your best to talk with me like [cg] 
[lg]:it means [cg] must talk with others like [aw]
[aw]:it means some word"(抬头) 你好！ 我是浮云[微笑]{好感度：1}"
 ```

其中的
[wm]指的是扮演的角色的外貌
[xg]指的是扮演的角色的性格
[at]不必更改（sfw）
[lg]是定义说话格式
[aw]则是对话示例

可以根据自己的实际需要来更改内容

效果展示:
![image](https://user-images.githubusercontent.com/125521033/234653085-c8f7a0c9-fca4-400b-86c3-668e1f5382a4.png)
![image](https://user-images.githubusercontent.com/125521033/234653257-fe741972-a410-4aea-8e69-7752c80cf36b.png)
![image](https://user-images.githubusercontent.com/125521033/234653313-b3654654-35e6-4549-9b2f-1ae6db485c82.png)

**Comment by TiansuoLi** [comment]
同prompt我发过去直接就是很抱歉🙏

**Reply by kongbaiwubai** [reply]
> 同提示我发过去直接接就是很抱憾🙏

我试了可以啊

**Reply by awaa-col** [reply]
不是100%成功，现在的对话格式还有点不稳定，我到时候再改改
也欢迎各路大神分享自己使用的提示词模板
